src/rviz/default_plugin/image_display.cpp

```diff
@@ -72,6 +72,8 @@ ImageDisplay::ImageDisplay() : ImageDisplayBase(), texture_()
                       this, SLOT(updateNormalizeOptions()));
 
   got_float_image_ = false;
+
+  mouse_click = new MouseClick();
```

Comment:
Nitpick: For deletion, you rely on RenderPanel being mouse_click's parent.
However, this is only set in `onInitialize()`!

Reply:
I am sorry but my cpp is a bit rusty. Like every sane person, I've moved to python a few years ago : - )

What do you suggest we do here?

Reply:
Pass a parent QObject* to MouseClick's constructor a forward this argument to the base class constructor (see below).